.github/workflows/pull-request.yml

```diff
@@ -24,6 +24,7 @@ jobs:
         pip install flake8 
         pip install build
         if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+        if [ -f requirements.txt ]; then pip install -r requirements-test.txt; fi
```

Comment:
Wouldn't it make more sense to put both in the same `if` statement if both are using the same condition?